src/i18n/en-motions.ts

```diff
@@ -0,0 +1,14 @@
+import { ColonyMotions } from '~types/index';
+
+const motionsMessageDescriptors = {
+  'motion.title': `{actionType, select,
+      ${ColonyMotions.MintTokensMotion} {Mint {amount} {tokenSymbol}}
+      other {Generic action we don't have information about}
+    }`,
+  'motion.type': `{actionType, select,
+      ${ColonyMotions.MintTokensMotion} {Mint Tokens}
+      other {Generic}
+    }`,
+};
```

Comment:
These we might just end up re-using from actions, but leave it like this for now.

We'll decide when we actually get to use it.